### PR TITLE
Add freebsd-powerpc64le sysroot

### PR DESCRIPTION
**Pull request by pkubaj**

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
Thank you!

This is unstripped. I'm 100% sure we don't need `lib80211.so.1` to build ClickHouse.
We also don't need C++ headers.

I will try to strip it and merge it.

**Comment by alexey-milovidov** [comment]
@pkubaj May I ask you to remove (most of) unneeded files?
You can do it by analogy with 
https://github.com/ClickHouse/sysroot/tree/master/freebsd-x86_64
and
https://github.com/ClickHouse/sysroot/tree/master/freebsd-aarch64

**Comment by alexey-milovidov** [comment]
@pkubaj please remove the unneeded files.

File: freebsd-powerpc64le/usr/include/c++/v1/__bit_reference [hunk 1]
```diff
@@ -0,0 +1,1305 @@
+// -*- C++ -*-
```
Comment (alexey-milovidov) [inline freebsd-powerpc64le/usr/include/c++/v1/__bit_reference:1]: We definitely don't need C++ headers.

**Comment by HeenaBansal2009** [comment]
@alexey-milovidov @pkubaj , I see we have ppc_wrappers  for porting x86_intrinsics in to PPC.
Do you think , it should be good to include as well .

for reference :
https://reviews.llvm.org/D65630

**Comment by alexey-milovidov** [comment]
@HeenaBansal2009 yes, let's send it in a subsequent pull request.

@pkubaj, we need to cleanup the files. Maybe I can do it on my own...

**Comment by HeenaBansal2009** [comment]
> @HeenaBansal2009 yes, let's send it in a subsequent pull request.
> 
> @pkubaj, we need to cleanup the files. Maybe I can do it on my own...

@alexey-milovidov , Sure I would like to take this up . can  you guide me from where I can get the licensed source files to be added under x86- .. That would be great

**Comment by alexey-milovidov** [comment]
@HeenaBansal2009 the files are located in the corresponding OS images, like ISO and Docker.

**Comment by alexey-milovidov** [comment]
@pkubaj let's cleanup the unneeded files.

**Pull request merged**